test/worlds/hydrodynamics.sdf.in

```diff
@@ -11,9 +11,24 @@
     <!-- prevent sinking -->
     <gravity>0 0 0</gravity>
 
+    <!-- Current Data -->
+    <plugin
+      filename="gz-sim-environment-preload-system"
+      name="gz::sim::systems::EnvironmentPreload">
+      <data>@CMAKE_SOURCE_DIR@/test/worlds/hydrodynamic_currents.csv</data>
```

Comment:
@arjo129 fyi: if both SDF and CSV files are sitting next to each other, you can use a relative path here.